delfin/drivers/fake_storage/__init__.py

```diff
@@ -131,6 +131,55 @@ def get_storage(self, context):
             'subscribed_capacity': subscribed
         }
 
+    @staticmethod
+    def get_spec(context):
```

Comment:
the reason to keep method name as such because with it driver can specify complete list of specifications other than resource_metrics. But yes currently it is limited to resource_metrics :)